concepticondata/concepticon.tsv

```diff
@@ -3885,3 +3885,6 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3899	ATTACKING	Warfare and hunting	To use violence with the intention to harm.	Action/Process	
 3900	CLUMSY	Motion	Moving carelessly or gracelessly, causing things to break or get knocked over.	Property	
 3901	TAMING	Animals	To domesticate an animal, bird, etc.; to make them used to living with and/or working for humans.	Action/Process	
+3902	TRUST (SOMEBODY)	Social and political relations	To have great confidence in someone that he or she is sincere, honest, and will not deliberately inflict any harm. 	Action/Process	
+3903	SYMPATHIZE	Social and political relations	To show understanding for somebody's problems or hurt feelings.	Action/Process	
+3904	ENCOUNTER (MEET)	Social and political relations	To meet someone or discover something, usually unexpectedly.	Action/Process	
```

Comment:
I'm not sure if we should add this concept since "to encounter" can be mapped to MEET and there are only a few instances with meet/encounter.

Reply:
Ok, but the two unmapped ones could also be the noun meaning "an encounter", right? And judging from the present mappings to MEET, e.g., _begegnen_, the meaning "encounter" is included.

Reply:
I see your point. But should we then also change the description of MEET? It might be better to open a separate issue and discuss this in detail.

Reply:
I opened an issue for that and started a discussion (#1077). I also interpret MEET as a hyperonym, implying both types of intentionality, while ENCOUNTER (MEET) is reserved for the unintentional encounter. In any case, even if there is just one list mapped to the new concept set, since we have already started discussing it and its structure in relation to other concepts, it would be fair to keep it for future use.

Reply:
Sounds good! I think you can go ahead and map it in the current list already ("encounter" is still unmapped).